docs/content/components/collection/selectlist/selectlist.mdx

```diff
@@ -25,29 +34,98 @@ A `SelectList` consists of a container element, with rows of data inside. The ro
 
 ## Usage
 
-The `SelectList` component is used to display a list of items that can be selected by the user. It supports both single and multiple selection modes, allowing users to choose one or more items from the list.
+Use `<SelectList>` when you need to present users with a set of options to choose from. It works well for configuration settings, filter selections, item management interfaces, and any scenario requiring user choice from a predefined set.
 
-Built with accessibility in mind, it supports full keyboard navigation, making it easy to interact with using only the keyboard.
+The component is particularly effective when options need descriptive text or visual elements to help users make informed decisions. For simple form inputs with fewer options, consider using [`<Select>`](/components/form/select/) or [`<RadioGroup>`](/components/form/radiogroup/) instead.
 
-### SelectList with multiple selection mode
+<GuidelineTiles>
+  <Do>
+    <Do.Description>
+      Use clear, descriptive labels that help users understand each option.
+    </Do.Description>
+  </Do>
+  <Do>
+    <Do.Description>
+      Provide visual hierarchy with images and descriptions when options benefit
+      from additional context.
+    </Do.Description>
+  </Do>
+</GuidelineTiles>
 
-If you want to allow users to select multiple items, you can set the `selectionMode` prop to `multiple`. This enables checkboxes for each item, allowing users to select or deselect multiple items at once.
+### Basic SelectList
 
-You can also provide interactive actions for each item by using `<SelectList.Action>`, such as buttons or links, to perform specific actions on the selected items.
+A simple `<SelectList>` with single selection mode allows users to choose one option from the list. Each item must have a unique `id` and a `textValue` for accessibility.
 
-<ComponentDemo
-  disableLabelWidth
-  file="./selectlist-multiple-selection.demo.tsx"
-/>
+<ComponentDemo file="./selectlist-basic.demo.tsx" />
 
-### Disabled behavior
+### SelectList with descriptions
 
-The `<SelectList>` lets you disable certain items, which is helpful if you don’t want users to select them. You can use the `disabledBehavior` prop to control what happens with those disabled items. In the example below, we use `disabledBehavior="selection"`. This means the disabled items can’t be selected and won’t show a selection indicator — but you can still use their actions (like buttons or links inside them).
+Add descriptions to provide additional context for each option. This helps users make informed decisions when selecting items.
 
-<ComponentDemo
-  disableLabelWidth
-  file="./selectlist-disabledbehavior.demo.tsx"
-/>
+<ComponentDemo file="./selectlist-with-description.demo.tsx" />
+
+### SelectList with images
```

Comment:
would use this example and also make it maybe multiple selection mode, so we don't have so many examples